### PR TITLE
Update net-keepalive to 1.3.1

### PR DESCRIPTION
**Pull request by Page-**
Connects-to: #1878 
Change-type: minor

### CODE REVIEW

**Comment by pdcastro** [comment]
I'm holding approval of this PR because of the CI error:
```
1) balena logs
       should reach the expected endpoints on a local device:
     AssertionError: expected [ 'This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:\n',

  'Error: No native build was found for platform=darwin arch=x64 runtime=node abi=64 uv=1 libc=glibc\n',
```

It looks like a proper error. If you try running `balena logs <ip-address>`, does it fail too? Both `npm install` and standalone zip (`npm run build:standalone`, `./build-bin/balena logs <ip-address>`) It is one of the commands that use `net-keepalive`. (The other command is `balena push <ip-address>`)

File: INSTALL.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ If you are a Node.js developer, you may wish to install the balena CLI via [npm]
 The npm installation involves building native (platform-specific) binary modules, which require
 some additional development tools to be installed first:
 
-* [Node.js](https://nodejs.org/) version 10 or 12 (version 14 is not yet fully supported)
+* [Node.js](https://nodejs.org/) version 10 (min **10.20.0**) or 12 (version 14 is not yet fully supported)
```
Comment (pdcastro) [inline INSTALL.md:96]: @Page-, I have added this note to the docs. Feel free to modify it.

**Comment by Page-** [comment]
@balena-ci rebase

**Comment by Page-** [comment]
@balena-ci retest

**Comment by pdcastro** [comment]
@Page-, I've got it working with an alternative PR: #1940 
Re commit message `Update net-keepalive to 1.3.1, adding support for nodejs 14`, I think _"adding support for nodejs 14"_ could be a bit misleading in the Changelog because some commands still don't support Node v14, even after updating `net-keepalive` and `pkg`. For example `os configure`, I think it was because of ext2 related modules.

**Pull request closed without merging**